load:
    x:
        wheel1: 
            p: 0.135 
            i: 0.002 
            d: 0.004

        wheel2:
            p: 0.135
            i: 0.002
            d: 0.004

        wheel3:
            p: 0.0675
            i: 0.002
            d: 0.004

        wheel4:
            p: 0.1025
            i: 0.002
            d: 0.004


    y:
        wheel1:
            p: 3.15
            i: 0.00001
            d: 0.001

        wheel2:
            p: 3.15
            i: 0.00001
            d: 0.001

        wheel3:
            p: 1.575
            i: 0.00001
            d: 0.001

        wheel4:
            p: 4.725
            i: 0.00001
            d: 0.001
            
    rotation:
        wheel1:
            p: 0.02
            i: 0.004
            d: 0.004

        wheel2:
            p: 0.02
            i: 0.004
            d: 0.004

        wheel3:
            p: 0.02
            i: 0.004
            d: 0.004

        wheel4:
            p: 0.02
            i: 0.004
            d: 0.004

unload:
    x:
        wheel1:
            p: 0.324
            i: 0.002
            d: 0.05

        wheel2:
            p: 0.324 
            i: 0.002
            d: 0.05

        wheel3:
            p: 0.162
            i: 0.002
            d: 0.05

        wheel4:
            p: 0.486
            i: 0.002
            d: 0.05
            

    y:
        wheel1:
            p: 5.76
            i: 0.008
            d: 0.003

        wheel2:
            p: 3 
            i: 0.008
            d: 0.003

        wheel3:
            p: 5.4
            i: 0.008
            d: 0.003

        wheel4:
            p: 1.92
            i: 0.008
            d: 0.003
            
    rotation:
        wheel1:
            p: 0.02
            i: 0.002
            d: 0.001

        wheel2:
            p: 0.02
            i: 0.003
            d: 0.001

        wheel3:
            p: 0.02
            i: 0.003
            d: 0.001

        wheel4:
            p: 0.02
            i: 0.003
            d: 0.001
